infrastructure/web/main.tf

```diff
@@ -0,0 +1,14 @@
+terraform {
+  required_providers {
+    google = {
+      source  = "hashicorp/google"
+      version = "4.50.0"
+    }
+  }
+}
+
+# Uses GOOGLE_CREDENTIALS environment variable
```

Comment:
I think it's my fault this comment is here. I know I put it on another file. Although I was wrong. This is not true so can we remove please?